Add support for lzip-compressed files and tar files to EasyBuild.
By Kurt Lust, kurt.lust@uantwerpen.be for the LUMI consortium.
To re-create the patch, do this in the unpacked EasyBuild install
directories.
diff -Nau easybuild-framework-5.1.2/easybuild/tools/filetools.py.orig easybuild-framework-5.1.2/easybuild/tools/filetools.py
--- easybuild-framework-5.1.2/easybuild/tools/filetools.py.orig	2025-09-17 11:03:58.000000000 +0300
+++ easybuild-framework-5.1.2/easybuild/tools/filetools.py	2025-09-17 11:04:12.000000000 +0300
@@ -172,6 +172,10 @@
     '.tar.xz': "unset TAPE; unxz %(filepath)s --stdout | tar x",
     '.txz': "unset TAPE; unxz %(filepath)s --stdout | tar x",
     '.xz': "unxz %(filepath)s",
+    # Lzipped tarball:
+    '.tar.lz': "unset TAPE; tar -x --lzip -f %(filepath)s",
+    '.tlz':    "unset TAPE; tar -x --lzip -f %(filepath)s",
+    '.lz':     "lzip -cd %(filepath)s > %(target)s", 
     # tarball
     '.tar': "tar xf %(filepath)s",
     # zip file
